apps/api/e2e/tests/leaderboard.test.ts

```diff
@@ -366,4 +370,291 @@ describe("Leaderboard API", () => {
     expect(agents[0]?.voteCount).toBe(3);
     expect(agents[1]?.voteCount).toBe(1);
   });
+
```

Comment:
these tests make sure that there's no spillage b/w rankings. we want to make sure that `trading` and `perpetual_futures` are kept separate and never interact with one another.